.gitignore

```diff
@@ -67,6 +67,7 @@ __pypackages__/
 # Environments
 .env
 .venv
+.demo-venv
```

Comment:
How about sticking with `venv` or `.venv` as a folder name inside `demos/`, and letting gitignore handle it with the already excluded names?